doc/source/cluster/config.rst

```diff
@@ -493,6 +501,35 @@ The extra options to pass to ``docker run``.
 * **Default:** ``[]``
 * **Update requires:** Restart
 
+
+.. _cluster-configuration-disable-automatic-runtime-detection:
+
+``docker.disable_automatic_runtime_detection``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If enabled, Ray will not try to use the NVIDIA runtime if GPUs are present.
+
+* **Required:** No
+* **Importance:** Medium
+* **Type:** Boolean
+* **Default:** ``False``
+* **Update requires:** Restart
+
+
+### disable_shm_size_detection
+.. _cluster-configuration-disable-shm-size-detection:
+
+``docker.disable_shm_size_detection``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If enabled, Ray will not automatically specify the size /dev/shm for the started container.
```

Comment:
what happens when ray does not automatically specifies it? what the default value will be?

Reply:
See my suggestion below